designdocs/privacy-design.md

```diff
@@ -0,0 +1,163 @@
+## Note
+The goal of this design is to protect user privacy when data from the WebXR Device API is made available to web sites.
+To accomplish this, mitigations (including but not limited to user consent) are outlined as either requirements or recommendations before allowing sites to access different types of data. An explanation of the mitigations and corresponding threat vectors can be found in the Background section.
+
+There is no distinction between different WebXR modes in this document. Data access requirements are the same whether the session is inline, immersive, or otherwise. If a user agent wants to avoid user consent for a particular type of session, the user agent should meet the requirements for all data available in that session. _For example, user agents that limit inline sessions to identity reference spaces only or apply necessary mitigations (such as data quantization) to eye-level and floor-level reference spaces may still offer them without first obtaining user consent._
+
+
+## Privacy Requirements
+### Conditions to create objects and expose data
+#### XRSession Creation
+
+The user agent must verify that all mandatory conditions are satisfied to ensure it can create an [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) and expose [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) data in response to a call to [XRRequestSession](https://immersive-web.github.io/webxr/#dom-xr-requestsession) within a given [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document).
+
+The **_mandatory conditions_** for [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) creation are the following:
+*   The document is a [responsible document](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-document) of a [secure context](https://w3c.github.io/webappsec-secure-contexts/#secure-contexts).
+*   [Currently focused area](https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context) belongs to a document whose origin is [same origin-domain](https://html.spec.whatwg.org/multipage/origin.html#same-origin-domain) with the origin of the given [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document).
```

Comment:
I'm supportive of that approach. 👍  The interesting thing is that this would only be possible for inline sessions because they're the only kind that can be created with all tracking disabled.  And I suspect that this not being called out explicitly was a big part of my initial confusion.